File: 0973-k-closest-points-to-origin/0973-k-closest-points-to-origin.py
class Solution:
    def kClosest(self, points: List[List[int]], k: int) -> List[List[int]]:
        results = []
        points = sorted(points, key= lambda x: sqrt(pow(x[0],2) + pow(x[1],2) ) )
        return points[0:k]

File: 0973-k-closest-points-to-origin/0973-k-closest-points-to-origin.cpp
bool compareDistance(vector<int> point1, vector<int> point2){
    return ( sqrt(pow(point1[0], 2) + pow(point1[1], 2) ) < sqrt(pow(point2[0], 2) + pow(point2[1], 2) ) );
}


class Solution {
public:

    vector<vector<int>> kClosest(vector<vector<int>>& points, int k) {
        vector<vector<int>> results{points};
        std::sort(results.begin(), results.end(), compareDistance);
        return vector<vector<int>>(results.begin(), results.begin() + k);
    }
};